markusapi/markusapi.py

```diff
@@ -335,6 +335,13 @@ def get_files_from_repo(self, assignment_id: int, group_id: int,
         if response[0] == 200:
             return response[2]
 
+    def create_folders(self, assignment_id: int, group_id: int, folder_path: str) -> List[str]:
```

Comment:
This should follow the same naming convention as other methods in this file:

`upload_folder_to_repo`